src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/AuthorizationApiServiceImpl.scala

```diff
@@ -117,10 +117,13 @@ final case class AuthorizationApiServiceImpl(
       subUnitType       <- institutionApi.subUnitType.traverse(TenantUnitType.fromValue(_).toFuture)
       onboardingData    <- selfcareV2ClientService.getOnboardingsInstitution(institutionApi.id, None)
       onboardingDataApi <- onboardingData.toApi.toFuture
-      resourceId        <- tenantProcessService
+      externalId =
+        if (institutionApi.origin == "IPA") institutionApi.subunitCode.getOrElse(institutionApi.originId)
+        else institutionApi.taxCode
```

Comment:
LGTM, but subunitCode is equal to originId in AOO/UO.
Let's keep this implementation for now, but we should check if it can be simplified.